functions/Get-DbaSqlBuildReference.ps1

```diff
@@ -25,6 +25,11 @@ Function Get-DbaSqlBuildReference {
 		Returns information about a build identified by  "12.00.4502" (which is SQL 2014 with SP1 and CU11)
 	
 	.EXAMPLE
+		Get-DbaSqlBuildReference -Build "12.00.4502" -Update
```

Comment:
Can you add 'update' in the .PARAMETER section, please?
Maybe some extensive explanation what happens when using it; does it overwrite a file?